tasks.py

```diff
@@ -216,8 +220,22 @@ def notify(ctx):
         username=env.get_credential('FEC_SLACK_BOT', 'fec-bot'),
     )
 
-def run_migrations(ctx, space):
-    print("\nMigrating database...")
-    jdbc_url = to_jdbc_url(os.getenv('FEC_MIGRATOR_SQLA_CONN_{0}'.format(space.upper())))
+@task
+def create_sample_db(ctx):
+    """
+    Load schema and data into the empty database pointed to by $SQLA_SAMPLE_DB_CONN
+    """
+
+    print("Loading schema...")
+    db_conn = os.getenv('SQLA_SAMPLE_DB_CONN')
```

Comment:
Is it required that we set this environment variable now?  Would it make sense to set a default?

Reply:
I would rather go with the Zen of Python - "Explicit is better than implicit." :-)